tests/unit/withOnyxTest.js

```diff
@@ -276,16 +272,12 @@ describe('withOnyx', () => {
                     },
                 })(ViewWithCollections);
                 render(<TestComponentWithOnyx3 onRender={onRender3} />);
-
-                return waitForPromisesToResolve();
-            })
-            .then(() => {
-                // When a single item in the collection is updated with mergeCollect()
-                Onyx.mergeCollection(ONYX_KEYS.COLLECTION.TEST_KEY, {
-                    test_1: {ID: 1, newProperty: 'yay'},
-                });
-                return waitForPromisesToResolve();
             })
+
+            // When a single item in the collection is updated with mergeCollect()
```

Comment:
I know this was there but
```suggestion
            // When a single item in the collection is updated with mergeCollection()
```